data/config/packages.json

```diff
@@ -1417,23 +1427,58 @@
 				]
 			}
 		},
-		"openrazer": {
+		"polychromatic": {
 			"packages": [
-				"openrazer-daemon"
+				"polychromatic", "openrazer-daemon"
 			],
 			"repos": {
 				"all": [
-					"ppa:openrazer/stable"
+					"ppa:polychromatic/stable", "ppa:openrazer/stable"
+				]
+			},
+			"post-install": "openrazer-setup"
+		},
+		"openrazer-setup": {
+			"script": "openrazer-setup"
+		},
+		"discord": {
+			"snap": [
+				"discord"
+			]
+		},
+		"obs-studio": {
+			"snap": [
+				"obs-studio"
+			]
+		},
+		"mangohud": {
+			"packages": [
+				"mangohud"
+			],
+			"repos": {
+				"all": [
+					"ppa:flexiondotorg/mangohud"
```

Comment:
There is no jammy packages here - do we know when or even if Wimpy will update for jammy?

Reply:
no info on that one ... Package was not on snap store
Should we drop these 2 for for now?

Reply:
replaced with universe